src/platform/imx8ulp/platform.c

```diff
@@ -25,7 +25,9 @@
 #include <sof/schedule/ll_schedule.h>
 #include <sof/schedule/ll_schedule_domain.h>
 #include <rtos/sof.h>
+#if CONFIG_TRACE
 #include <sof/trace/dma-trace.h>
```

Comment:
Feel free to just drop the inclusion of `dma-trace.h` altogether if not too much of a hassle. 8ulp doesn't use trace anymore.

Reply:
Ack @LaurentiuM1234 , works for me. I was testing all the upstream imx builds and this indeed was still required to be there.